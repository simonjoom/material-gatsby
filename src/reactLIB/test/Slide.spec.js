import React from 'react';
import { shallow } from 'enzyme';
import Slide from '../src/Slide';

describe('<Slide />', () => {
  const wrapper = shallow(<Slide src="#" />);

  test('renders', () => {
    expect(wrapper).toMatchSnapshot();
  });

  test('should render a Slide', () => {
    expect(wrapper.childAt(0).is('img')).toEqual(true);
    expect(wrapper.childAt(0).is({ src: '#' })).toEqual(true);
  });
});
