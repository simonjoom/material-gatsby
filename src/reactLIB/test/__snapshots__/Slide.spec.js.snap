// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Slide /> renders 1`] = `
<li>
  <img
    src="#"
  />
</li>
`;
